adding_species_manuscript.md

```diff
@@ -107,133 +126,120 @@ since it is these patterns of polymorphism themselves that we want to compare to
+linkage (correlations in transmission and inheritance between adjacent parts of the chromosome) decreases the amount of 
+variation in a chromosome simulated as a single continuous stretch, compared to simulating 
+multiple smaller fragments that add up to the same length. That is, treating a 100Mb chromosome as 100 independent 1Mb chunks of chromosome
+artificially increases the amount of independence in the data. This may be misleading if the scale of linkage is long:
+for instance, results on simulated human chromosome 22 are often very noisy due to a long stretch of near-zero recombination (TODO: CITE - @petrelharp?).
+These effects would not have been evident if the chromosome had been broken the chromosome into many independent pieces for simulation. 
+
+Second, linked selection (the effect that natural selection has on patterns of inheritance, and hence genetic diversity,
+at nearby locations on the genome) strongly affects patterns of genetic variation. 
 For instance, since many types of selection reduce diversity nearby,
 one might expect chromosome ends to have increased diversity as they have less flanking sequence
-on which selection might act.
-(In practice, other factors are at play as well.)
-Without using a genetic map,
+on which selection might act (though other factors are in play as well). Without using a genetic map,
 we can't look at the effect of linked selection on the correlation between recombination rate
-and genetic diversity, that is commonly observed in practice (CITE).
+and genetic diversity that is commonly observed in practice (CITE).
 The scale over which linked selection has an effect in practice
 can differ greatly depending on the species and the context,
 and the actual extent is unknown.
-Even in large-population-size species,
-recent selective sweeps (e.g., insecticide resistance XXX)
+Even in species with large population sizes,
+recent selective sweeps (e.g., insecticide resistance TODO:CITE)
 can cause large haplotypes to reach high frequency.
-Another practical consideration is demographic realism:
-the genetic load in a simulation of a small segment of genome with deleterious mutations
+
+Third, linkage has the potential to affect demographic realism:
+the genetic load in a simulation of a small segment of chromosome with deleterious mutations
 will necessarily be less than that in whole chromosome;
 this makes it easy to simulate with unrealistically high levels of load
 without realizing it if small segments are used.
+
+However it is much faster to simulate many small chunks of genome, in part because it effectively parallelizes the simulation.
 It's probably safe to say that independently simulating small segments
 will not introduce serious problems in some situations, but could cause serious biases in others.
 The degree to which it is important to include linked selection in whole-chromosome simulations
 is, we think, still a major open question in the field.
```

Comment:
I don't like wishy-washy statements like "we think". We wouldn't write that sentence if we didn't think it!